app/listeners/webhook_listener.rb

```diff
@@ -3,13 +3,17 @@ def message_created(event)
     message = extract_message_and_account(event)[0]
     inbox = message.inbox
 
-    return unless message.reportable? && inbox.webhook.present?
+    return unless message.reportable?
```

Comment:
Shouldn't we do the rest of the things in the background job? Otherwise this queries would run as a part of message_creation event right? @sony-mathew @sojan-official Thoughts?

Reply:
Agree. There is a better place to fix this.  Webhook/Reporting/Bot listener uses Asyncdispatcher to dispatch the event. The original idea of this async dispatcher to dispatch event asynchrsly which we are not doing it now. If we do that the all the after event activities like reporting/webhook/bot will be done in a defered way.

Reply:
@subintp I am aligned to making the change at Asyncdispatcher.